### PR TITLE
Feature/ntlm auth wrapper

### PR DESCRIPTION
**Pull request by louismunro**
Add a utility ("bin/ntlm_auth_wrapper") to log ntlm authentication times.

### CODE REVIEW

**Comment by fdurand** [comment]
Arch dependent, cannot be merge like that.
You must add packaging change

File: src/ntlm_auth_wrap.c [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/* A wrapper around ntlm_auth to log arguments and
```
Comment (extrafu) [inline src/ntlm_auth_wrap.c:1]: Please add the copyright/license header like in perl files.

Comment (louismunro) [inline src/ntlm_auth_wrap.c:1]: Done. Added the header from pfcmd.c.

**Comment by jrouzierinverse** [comment]
Please rebase
Add compile step to debian/rules and addons/packages/packetfence.spec

**Comment by dwlfrth** [comment]
I think that should be merged and maybe even used by default in our call of the mschap module.
That adds some interesting logging.

**Comment by dwlfrth** [comment]
What's my status ?

**Comment by louismunro** [comment]
I am working on adding CLI options to it for things like log level and even StatsD support.

Also it needs to be compiled for each architecture so there's packaging involved.

---

LM

**Comment by louismunro** [comment]
Closing. Will be part of the dashboard rework in 5.1.

**Pull request closed without merging**